main.py

```diff
@@ -13,93 +12,45 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# [START app]
+import datetime
 import logging
-import psycopg2
+import os
+import json
 
 from flask import Flask
-from flask import jsonify
-from flask_cors import CORS
```

Comment:
We still need flask-cors to allow CORS because front-end development and production env will be of different origin.